assets/dev/js/editor/regions/navigator/navigator.js

```diff
@@ -92,7 +92,9 @@ export default class extends BaseRegion {
 
 		this.$el.draggable( this.getDraggableOptions() );
 
-		this.$el.resizable( this.getResizableOptions() );
+		setTimeout( () => {
```

Comment:
Why it happen? and what about `this.$el.draggable`?

Reply:
@iNewLegend ?

Reply:
bug in JQUERY UI as i understand.

Reply:
What about  `draggable`?

Reply:
the conflict is between the both.

Reply:
Add an explanation.